src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/WebScenarioTests.cs

```diff
@@ -34,7 +34,8 @@ private static IEnumerable<TestScenario> GetScenarios()
         }
 
         yield return new(nameof(WebScenarioTests), DotNetLanguage.CSharp, DotNetTemplate.Razor,         DotNetActions.Build | DotNetActions.Run | DotNetActions.Publish);
-        yield return new(nameof(WebScenarioTests), DotNetLanguage.CSharp, DotNetTemplate.BlazorWasm,    DotNetActions.Build | DotNetActions.Run | DotNetActions.Publish);
+        // Disable building blazorwasm tests until https://github.com/dotnet/source-build/issues/2946 is addressed
```

Comment:
Nit: leave the code and disable it with a `/* ... */` comment.  It makes it easier to re-enable.